docker/drama-free-django/_build.sh

```diff
@@ -29,7 +29,7 @@ yum install -y gcc git python27
 source /opt/rh/python27/enable
 
 pip install -U pip
-pip install -U git+https://github.com/cfpb/drama-free-django.git
+pip install -U git+https://github.com/cfpb/drama-free-django.git@magic-removal-branch
```

Comment:
Once https://github.com/cfpb/drama-free-django/pull/25 is merged, don't forget to revert this.  :)